Application/EdFi.Ods.AdminApi.AdminConsole/EdFi.Ods.AdminApi.AdminConsole.csproj

```diff
@@ -1,21 +1,63 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>

```

Comment:
Please change to `True`. Or, if that introduces too many warnings to fix right away in this ticket, please create a new ticket to put this to the proper setting, and then we need to prioritize addressing this soon.

Reply:
I'll create a ticket to address the errors.

Reply:
[ADMINAPI-1161](https://edfi.atlassian.net/browse/ADMINAPI-1161)

Reply:
Thank youf